### ISSUE TITLE
Associate a log with a recipe

### ISSUE THREAD

**Issue by jclif** [Roastero/Openroast#34]
It'd be interesting if a given log (I like the term batch) could be associated with a given recipe. I can imagine wanting to experiment with a given recipe so that other salient attributes like the amount and type of bean can be tested and the results compared.

**Comment by lodge93** [comment]
I agree! We had started a log tab a few months ago, but I have since been focusing on making the project more maintainable. I would like to get the project to a point here shortly where myself or other developers can add such features without too much fuss.

**Comment by int3ll3ct** [comment]
All this IMHO - not an unreasonable request, but there's a danger of attempting to re-create Artisan in the process.  In my mind, we should be keeping Openroast focused on doing the basics well until we build an Artisan interface for the SR700, and allow super-users to geek out on that.
